### PR TITLE
fixes #17: use PIDs as Swift identifiers instead of filename

### LINKED ISSUE #17: Swift ID's based on filename instead of repository ID #6
Can the Swift stored objects have IDs that directly correspond to the upstream id instead of the filename link?

Currently, Swift objects are stored with IDs based on the filename (with : replaced with _) : cwrc_ff906b06-78f4-4b87-a488-4e71492289e0

Whereas, the repository PID has a : : cwrc:ff906b06-78f4-4b87-a488-4e71492289e0

This makes matching harder between UAL and CWRC required the knowledge to translate between the slightly different ids.

### PR DESCRIPTION
**Pull request by jefferya**
fixes #17 

changes the Swift ID to match the CWRC PID

### CODE REVIEW

File: cwrc_audit_report.rb [hunk 1]
```diff
@@ -111,7 +111,7 @@ module CWRCPerserver
     cwrc_pid = cwrc_obj['pid']
     cwrc_mtime = cwrc_obj['timestamp']
     # account for the CWRC PID ':' replaced with "_" in the Swift ID
```
Comment (pgwillia) [inline cwrc_audit_report.rb]: Comment could be removed. No longer the case.

**Review by pgwillia — CHANGES_REQUESTED** [review]
Agree that the conversion to from isn't necessary and potentially confusing.  I noticed that you missed the tests:

https://github.com/ualbertalib/cwrc_preservation/blob/master/cwrc_preserver_test.rb#L92
and
https://github.com/ualbertalib/cwrc_preservation/blob/master/cwrc_preserver_test.rb#L74

You would have to update the vcr recordings as well.

**Review by weiweishi — DISMISSED** [review]
This makes so much more sense. Thanks for spotting and fixing this!

File: cwrc_audit_report.rb (hunk 1 above)
Comment (jefferya) [reply cwrc_audit_report.rb]: Nice catch, thx!

**Review by pgwillia — APPROVED** [review]

**Pull request merged**